### PR TITLE
adding base tab CSS

### PR DESCRIPTION
**Pull request by williamdingwall**
@saralohr I'm curious to here your opinion on the data attributes to control  functionality of the tabs... this is how it works within classic. Thinking it might be good to some of the functionality for classic for development in places like registration, maybe account, potentially other places.

### CODE REVIEW

File: src/css/components/index.scss [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 @import "StepNav";
-@import "Panel";
-@import "Loader";
-@import "Divider";
-@import "Progress";
 @import "SummaryList";
+@import "Tabs";
+@import "Toggle";
 @import "Tooltip";
-@import "Feedback";
```
Comment (saralohr) [inline src/css/components/index.scss]: Thanks for alphabetizing these.

File: src/css/components/Tabs.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    display: block;
+    width: 100%;
+    height: $border-width-large;
+    position: absolute;
+    left: 0;
+    bottom: 0 - $border-width-large;
+    background: transparent;
+  }
+
+  &.js-active,
```
Comment (saralohr) [inline src/css/components/Tabs.scss]: This would be `is-active` according to SUIT conventions.

Comment (williamdingwall) [reply src/css/components/Tabs.scss]: Oh right, yeah I'll update that - here's the ui-patterns PR https://github.com/teamsnap/ui-patterns/pull/56

File: src/css/components/index.scss (hunk 1 above)
Comment (williamdingwall) [reply src/css/components/index.scss]: no problem!

**Pull request merged**